DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>I'm not sure I understood this point. Is this referring to reading in the catalog in the above cell, or below cell (or is this not actually demonstrated?) Maybe this section 4.2 needs an introductory sentence explaining more about what is about to be demonstrated where it could be explained before doing anything. Also the introductory sentence could say i.e. how is this different from the user-uploaded table from section 2? is it just a different format? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>The only difference between the tables in Sections 2,3 and the table in this section is that there are string columns. I am also not sure what's the requirement for the uploading table format, as I responded to one of your earlier comments. I figured out a solution based on the error msg when trying to upload a table including columns in the unicode format, but without a full understanding what's going on here.</p><p> </p>